Slim/Web/Cometd.pm

```diff
@@ -638,16 +640,16 @@ sub sendResponse {
 
 	$out ||= [];
 
-	# Add any additional pending events
-	push @{$out}, ( $manager->get_pending_events( $httpClient->clid ) );
+	if ($httpResponse) {
+		# Add any additional pending events
+		push @{$out}, ( $manager->get_pending_events( $httpClient->clid ) );
 
-	# Add special first event for /meta/(re)connect if set
-	# Note: calling first_event will remove the event from httpClient
-	if ( my $first = $httpClient->first_event ) {
-		unshift @{$out}, $first;
-	}
+		# Add special first event for /meta/(re)connect if set
+		# Note: calling first_event will remove the event from httpClient
+		if ( my $first = $httpClient->first_event ) {
+			unshift @{$out}, $first;
+		}
```

Comment:
Please describe what you're doing here, why you're doing it, and why it's working without breaking things.

Reply:
As far as I've been able to determine, `$httpResponse` will always eval to true here for HTTP clients and always to false for CLI clients. The original code that was outside the conditional would fail for CLI clients and result in a response never being returned to the CLI plugin and subsequent CLI commands would return a Busy! error.

This change moves the existing HTTP client specific code under the conditional for `$httpResponse` so it's not executed for CLI clients.